hikari/webhooks.py

```diff
@@ -855,11 +855,21 @@ class ChannelFollowerWebhook(PartialWebhook):
         This will be `None` when received within an audit log.
     """
 
-    source_channel: channels_.PartialChannel = attr.field(eq=False, hash=False, repr=True)
-    """The partial object of the channel this webhook is following."""
+    source_channel: undefined.UndefinedOr[channels_.PartialChannel] = attr.field(eq=False, hash=False, repr=True)
```

Comment:
why not just Optional

Reply:
Just marks a difference between the channel actually existing and Discord just not wanting to send it VS Discord not sending it because its deleted (if that could be a possibility in the future)

Reply:
this is specifically the *channel follow* webhook type though

Reply:
I might be missing something, but not really sure what you mean? I know that its dumb that Discord dont send it, but thats how things are:

![image](https://user-images.githubusercontent.com/29100934/216039202-3a48f0eb-f47a-42e1-8435-e801f5e93fbf.png)

https://discord.com/developers/docs/resources/webhook#webhook-object-webhook-structure

Reply:
as in this webhook type has to be linked to a guild channel so that future case you specified shouldn't happen but also in the case that it does discord wouldn't distinguish between it and that other case in the model so that reason for using Undefined over None there just doesn't really work

Reply:
Fixed